rov/rov.py

```diff
@@ -109,3 +113,43 @@ def run(lock, data):
         except Exception as e:
             print "Exception: %s" % e
             print traceback.format_exc()
+
+"""This main loop will only run if ./scotty run --rov is called.
+    This main loop will be used to run the ROV without having to
+    run the flask server."""
+if __name__ == "__main__":
```

Comment:
This doesn't really make sense to have in rov's main. Why can't this hook into the other existing test stuff? It can just import the `ROV` object. ROV's main was meant for just running the ROV loop, which was sometimes useful (Mainly just to start it up and make sure nothing crashed I guess, so for testing -- meaning it maybe shouldn't have a main at all)